core/src/rigidbody.cpp

```diff
@@ -19,6 +25,9 @@ void RigidBody::on_awake() {
     if (get_shape_from_shape()) {
         this->m_shape = get_shape_from_shape();
     }
+    if (get_aggregate_from_aggregate()) {
+        this->m_aggregate = get_aggregate_from_aggregate();
+    }
```

Comment:
```suggestion
	this->m_shape = get_shape_from_shape();
    this->m_aggregate = get_aggregate_from_aggregate();
```

If it is returning a `nullptr` we can keep it that way, if not, please change those functions to return a `nullptr` on a not valid value.